server/fleet/hosts.go

```diff
@@ -316,6 +316,9 @@ type Host struct {
 	// The boolean is based on information ingested from the Apple DEP API that is stored in the
 	// host_dep_assignments table.
 	DEPAssignedToFleet *bool `json:"dep_assigned_to_fleet,omitempty" db:"dep_assigned_to_fleet" csv:"-"`
+
+	// LastRestarted is a UNIX timestamp that indicates when the Host was last restarted.
+	LastRestarted time.Time `json:"last_restarted" db:"last_restarted"`
```

Comment:
Should this be a pointer to account for when `last_restarted` is unknown? (related to https://github.com/fleetdm/fleet/pull/14878/files#r1383700222)

Reply:
Oh I see. I think I misunderstood your original comment; I thought we had to return _some_ value, even if it's the zero value for `time.Time`. I can make this a pointer, yes.